tests/cli/test_build.py

```diff
@@ -46,6 +46,36 @@ def test_build_by_tags():
     )
     cmd.build()
 
+    #  testing multiple tags as comma seperated list:  buildtest build --tags fail,python --tags network
+    cmd = BuildTest(
+        configuration=configuration,
+        tags=["fail,python", "network"],
+        buildtest_system=system,
+    )
+    cmd.build()
+
+    cmd = BuildTest(
+        configuration=configuration,
+        tags=[",,"],
+        buildtest_system=system,
+    )
+    with pytest.raises(SystemExit):
+        cmd.build()
+
+    cmd = BuildTest(
+        configuration=configuration,
+        tags=["python,"],
```

Comment:
now that i think about this, lets change this to something like this, which should capture all the edge cases.  `tags=["  ,python,  fail,   ,"].

This way we can remove the following statement

```
    cmd = BuildTest(
        configuration=configuration,
        tags=[",python"],
        buildtest_system=system,
    )
    cmd.build()
```

There are few edge cases that i can think of 
1. comma at start `buildtest bd -t ,python`
2. comma at end `buildtest bd -t python,`
3. comma with spaces `buildtest bd -t python,  fail,network  ,`